java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -300,4 +297,57 @@ void testReplaceNullsFailsOnNullScalar() {
       });
     }
   }
+
+  @Test
+  void testIntegerQuantiles() {
+    assumeTrue(Cuda.isEnvCompatibleForTesting());
+    int[] approxExpected = {-1, 1, 1, 2, 7};
+    double[][] exactExpected = {
+        {-1.0,   1.0,   1.0,   2.0,   7.0},
+        {  -1,     1,     1,     2,     7},
+        {  -1,     1,     1,     2,     7},
+        {-1.0,   1.0,   1.0,   2.0,   7.0},
+        {  -1,     1,     1,     2,     7}};
+    double[] q = {0.0, 0.25, 0.33, 0.5, 1.0};
+
+    try (ColumnVector cv = ColumnVector.fromBoxedInts(7, 0, 3, 4, 2, 1, -1, 1, 6)) {
```

Comment:
Can we do this with an even number of entries so there is some kind of a difference in the quantile calculations?